layers/+lang/java/layers.el

```diff
@@ -10,3 +10,4 @@
 ;;; License: GPLv3
 
 (configuration-layer/declare-layer 'groovy)
+(configuration-layer/declare-layer 'lsp)
```

Comment:
We should not require lsp layer in general, there may still be people wanting to use the non lsp java backends.
When lsp is not loaded neither the lsp-java package would be loaded so this should not make any problems but it will protect non lsp guys from having to install a bunch of non used packages with this layer.

Reply:
Thanks for the detailed explanation! Updated in latest diff. Just wonder whether the `message` should be `warn`.